dashboard-ui/src/components/SurveyResults/surveyResults.jsx

```diff
@@ -213,7 +215,7 @@ export function SurveyResults() {
                 }
                 for (const x of Object.keys(obj)) {
                     const res = obj[x];
-                    if (res?.scenarioIndex === selectedScenario) {
+                    if (res?.scenarioIndex == selectedScenario) {
```

Comment:
Was the triple equal removed on purpose?

Reply:
Yes, for the type conversion. Although I could cast this earlier and keep the triple equal

Reply:
Yeah, probably better to keep types the same